scrapers/scrape_bs.py

```diff
@@ -1,8 +1,28 @@
 #!/usr/bin/env python3
+import requests
+from bs4 import BeautifulSoup
+
 
 import scrape_common as sc
+import utils.bs as bs
+
+def text_from_url(url: str) -> str:
+    req = requests.get(url)
+    html = BeautifulSoup(req.content, "html.parser")
+
+    # I dont really know what I am doing here...
+    # But seems to be working.
+    # The goal is to extract the main text from the URL.
+    texts = []
+    for p in html.find_all("p")[1:-8]:
+        text = p.get_text()
+        if len(text.split(" ")) > 5:
+            texts.append(text)
+    return " ".join(texts)
 
 print('BS')
+sc.timestamp()
```

Comment:
Put (move) this `sc.timestamp()` just after `text = text_from_url(f'https://www.gd.bs.ch/{URL}')` on line 38.